src/main/java/talkabot/TalkaBot.java

```diff
@@ -0,0 +1,166 @@
+package talkabot;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+
+import talkabot.exceptions.InvalidEditException;
+import talkabot.exceptions.InvalidScheduleException;
+import talkabot.exceptions.NoInputException;
+import talkabot.exceptions.UnknownInputException;
+import talkabot.exceptions.TalkaBotException;
+import talkabot.task.Deadline;
+import talkabot.task.Event;
+import talkabot.task.Task;
+import talkabot.task.TaskList;
+import talkabot.task.ToDo;
+
+/**
+ * TalkaBot Class handles the running of Talk-a-Bot
+ * and responds according to user input.
+ */
+public class TalkaBot {
+
+    private Ui ui;
+    private TaskList taskList;
+    private Storage storage;
+
+    /**
+     * Constructor for TalkaBot class.

```

Comment:
Should this be "Constructs a TalkaBot class..." instead? 
```suggestion
     * Constructs a TalkaBot class...

```